splunk_add_on_ucc_framework/commands/build.py

```diff
@@ -631,3 +634,15 @@ def generate(
         from additional_packaging import additional_packaging
 
         additional_packaging(ta_name)
+    logger.info(f'openapi={openapi}')
+    if os.path.isfile(config_path) and openapi:
```

Comment:
Do we need `if os.path.isfile(config_path)` down here in the end? Or can we just check for `openapi` variable?

Reply:
If someone sends --openapi=1, but there is no globalConfig (.conf only TA), we'd have exception thrown
I think that our goal is to have openapi enabled by default. Condition like this will allow us to avoid the exceptions